test/functional/wallet_multiwallet.py

```diff
@@ -65,7 +65,7 @@ def wallet_file(name):
             assert_equal(os.path.isfile(wallet_file(wallet_name)), True)
 
         # should not initialize if wallet path can't be created
-        exp_stderr = "boost::filesystem::create_directory: (The system cannot find the path specified|Not a directory):"
+        exp_stderr = "boost::filesystem::create_director"
```

Comment:
Please confirm this is not a TYPO?
 create_director

Reply:
this is indeed not a typo. Boost changed the text of the error message starting in version 1.78. so, to maintain compatibility with earlier and newer versions of Boost, a partial text match is now used. `create_director` will match the old (`create_directory`) as well as the new (`create_directories`) string.